src/binding/binding.cpp

```diff
@@ -30,23 +32,23 @@ NAPI_MODULE_INIT() {
 	napi_create_string_utf8(env, rocksdb::GetRocksVersionAsString().c_str(), NAPI_AUTO_LENGTH, &version);
 	napi_set_named_property(env, exports, "version", version);
 
-	[[maybe_unused]] int refCount = ++moduleRefCount;
```

Comment:
I am curious about this change, as I have been using the same shorthand incrementation/decrementation pattern you are deleting. I thought https://en.cppreference.com/w/cpp/atomic/atomic/operator_arith.html explicitly guarantees `++moduleRefCount` and `moduleRefCount.fetch_add(1) + 1` are identical expressions (and same is true that `--moduleRefCount` and `moduleRefCount.fetch_sub(1) - 1` are guaranteed to be identical) because  `std::atomic` does operator overloading of `++` and `--` to ensure they are atomic. Do you know how they differ or sure they differ?